addons/l10n_mx/models/account.py

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+# Copyright 2016 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import re
+from odoo import models, api, fields, _
+
+
+class AccountAccount(models.Model):
+    _inherit = 'account.account'
+
+    @api.multi
+    def assign_account_tag(self):
+        """Based on account code will be assigned by default the corresponding
```

Comment:
This sounds like a hack to auto-assign tags that doesn't work well with all the localizations. You should avoid this.